app/en/home/examples/page.mdx

```diff
@@ -0,0 +1,124 @@
+    href="https://github.com/dforwardfeed/slack-AIpodcast-summaries"
+    image="/images/logo/arcade.png"
+    tags={["Python", "Langchain", "Slack", "Weaviate"]}
+    blank
+  />
+  <SampleAppCard
+    title="Baseball dugout"
+    description="An AI-powered scout that researches players, generates reports, and shares them via email—all from a single conversation."
+    href="https://github.com/ArcadeAI/baseball-dugout"
+    image="/images/logo/arcade.png"
+    tags={["JavaScript", "GitHub", "Email"]}
+    blank
+  />
+  <SampleAppCard
+    title="Arcade Custom Verifier Next"
+    description="Custom user verification implementation for Next.js applications using Arcade."
+    href="https://github.com/ArcadeAI/arcade-custom-verifier-next"
+    image="/images/logo/arcade.png"
+    tags={["TypeScript", "Next.js"]}
+    blank
+  />
+  <SampleAppCard
+    title="Agent Kitchen Sink"
+    description="Comprehensive TypeScript example showcasing various agent capabilities and integrations."
+    href="https://github.com/ArcadeAI/agent-kitchen-sink-typescript"
+    image="/images/logo/arcade.png"
+    tags={["TypeScript"]}
+    blank
+  />
+  <SampleAppCard
+    title="Agency Tutorial with Stytch"
+    description="Tutorial demonstrating agency patterns with Stytch authentication integration."
+    href="https://github.com/ArcadeAI/agency-tutorial-stytch"
+    image="/images/logo/arcade.png"
+    tags={["TypeScript", "Stytch"]}
+    blank
+  />
+  <SampleAppCard
+    title="Framework Showdown"
+    description="Comparison and examples of different AI agent frameworks and their capabilities."
+    href="https://github.com/ArcadeAI/framework-showdown"
+    image="/images/logo/arcade.png"
+    tags={["TypeScript"]}
+    blank
+  />
+  <SampleAppCard
+    title="Agent Templates"
+    description="Collection of ready-to-use agent templates for various use cases and patterns."
+    href="https://github.com/ArcadeAI/agent-templates"
+    image="/images/logo/arcade.png"
+    tags={["TypeScript"]}
+    blank
+  />
+  <SampleAppCard
+    title="Megaforce"
+    description="Powerful multi-agent coordination system for complex task orchestration."
+    href="https://github.com/ArcadeAI/megaforce"
+    image="/images/logo/arcade.png"
+    tags={["TypeScript"]}
+    blank
+  />
+  <SampleAppCard
+    title="CLI Agent Template"
+    description="Command-line interface template for building agents with terminal interactions."
+    href="https://github.com/ArcadeAI/cli-agent-template"
+    image="/images/logo/arcade.png"
+    tags={["TypeScript"]}
+    blank
+  />
+  <SampleAppCard
+    title="OpenAI SDK MCP Gateway"
+    description="Gateway implementation connecting OpenAI SDK with MCP (Model Context Protocol) servers."
+    href="https://github.com/ArcadeAI/openaisdk-mcpgateway"
+    image="/images/logo/arcade.png"
+    tags={["TypeScript", "OpenAI"]}
+    blank
+  />
```

Comment:
Same question about metadata and the design system repo, where should these live? @sdserranog